scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -171,7 +171,7 @@ object ScroogeSBT extends AutoPlugin {
     scroogeDefaultJavaNamespace := "thrift",
     scroogeThriftSourceFolder <<= (sourceDirectory) { _ / "thrift" },
     scroogeThriftExternalSourceFolder <<= (target) { _ / "thrift_external" },
-    scroogeThriftOutputFolder <<= (sourceManaged) { identity },
+    scroogeThriftOutputFolder in Compile := (sourceManaged in Compile).value / "thrift",
```

Comment:
As the comments above this section point out the configuration here should be in both Compile *and* test configuration. Hence this is a breaking change since it sets the output folder in the Compile configuration only. This probably needs to be modified to be:

```
scroogeThriftOutputFolder in Compile := (sourceManaged in Compile).value / "thrift",
scroogeThriftOutputFolder in Test := (sourceManaged in Test).value / "thrift",
```